tests/test_format.py

```diff
@@ -61,6 +61,19 @@ def test_strip_comments_single(self):
               'from foo--comment\nf'
         res = sqlparse.format(sql, strip_comments=True)
         assert res == 'select a\nfrom foo\nf'
+        # SQL-Hints have to be preserved
```

Comment:
I'd make separate tests for the SQL hints. Same code, just a separate function.

Reply:
Done.